src/app/metadata/features/dataset-details/dataset-details.component.html

```diff
@@ -467,7 +467,9 @@ <h4 class="inline-flex items-center">
                   File hash
                 </th>
                 <td mat-cell *matCellDef="let file">
-                  {{ file.file_information?.sha256_hash }}
+                  <abbr title="{{ file.file_information?.sha256_hash }}">{{
+                    file.file_information?.sha256_hash | shortHash
+                  }}</abbr>
```

Comment:
Would be nice to have an option to copy the full sha. GitHub offers this for the commit sha in several places, using a small "copy" button. Alternatively, a right-click menu option would also do. Should not be too difficult to implement. Could then be an ordinary span.